reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb

```diff
@@ -459,7 +459,7 @@
     "                             instance_type=instance_type,\n",
```

Comment:
<p>Do all of our RL notebooks support python 3.7 or above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
They should, we don't use any deprecated Python 3 functions.